spec/factories/miq_dialog.rb

```diff
@@ -37,4 +37,14 @@
     name        "miq_provision_configured_system_foreman_dialogs"
     dialog_type "MiqProvisionConfiguredSystemWorkflow"
   end
+
+  factory :miq_dialog_aws_provision, :parent => :miq_dialog do
+    name        "miq_provision_amazon_dialogs_template"
+    dialog_type "MiqProvisionWorkflow"
+
+    content do
+      YAML.load_file Rails.root.join("product", "dialogs", "miq_dialogs",
+                                     "miq_provision_amazon_dialogs_template.yaml")
```

Comment:
The YAML file contains the miq_dialog definition, if you just want the content you need to access just that hash value:

```ruby
(YAML.load_file Rails.root.join("product", "dialogs", "miq_dialogs",
                                "miq_provision_amazon_dialogs_template.yaml"))[:content]
```